stream.js

```diff
@@ -17,7 +17,7 @@ function WebSocketStream(target, protocols, options) {
   if (protocols && !Array.isArray(protocols) && 'object' === typeof protocols) {
     // accept the "options" Object as the 2nd argument
     options = protocols
-    protocols = null
+    protocols = []
```

Comment:
This is no longer needed. It was a regression bug which has been fixed in `ws@2.2.3`, sorry about that.